StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1516UnitTests.cs

```diff
@@ -372,6 +372,38 @@ public class Foo
             await this.VerifyCSharpFixAsync(testCode, fixedCode);
         }
 
+        [Fact]
+        public async Task TestThatCodeFixWorksOnFieldsAdjacentToMultiLineFields()
+        {
+            string testCode = @"using System;
+
+public class Foo
+{
+    private string experiment =
```

Comment:
@vweijsters Please file a separate issue for that if you'd like us to consider a change from the current behavior. It would take more discussion.